engines/bops_submissions/spec/services/application/parsers/proposal_parser_spec.rb

```diff
@@ -27,7 +27,7 @@
 
     context "with missing polygon" do
       let(:params) do
-        fixture = ActionController::Parameters.new(json_fixture("files/applications/PT-10087984.json")).deep_dup
+        fixture = json_fixture("files/applications/PT-10087984.json").deep_dup
```

Comment:
Feels like this should be a different fixture file

Reply:
Yeah, I wanted to avoid changing the test logic too much though.